packages/playwright/src/takeSnapshot.ts

```diff
@@ -1,10 +1,10 @@
 import type { Page, TestInfo } from '@playwright/test';
 import { readFileSync } from 'fs';
 import { dedent } from 'ts-dedent';
-import type { elementNode } from 'rrweb-snapshot';
+import type { elementNode } from '@chromaui/rrweb-snapshot';
 import { logger } from '@chromatic-com/shared-e2e';
 
-const rrweb = readFileSync(require.resolve('rrweb-snapshot/dist/rrweb-snapshot.js'), 'utf8');
+const rrweb = readFileSync(require.resolve('@chromaui/rrweb-snapshot'), 'utf8');
```

Comment:
Since `2.0.0-alpha.7`, `rrweb-snapshot` added an `exports` field in their `package.json` file, so we no longer have to rummage into the file structure here to pull out the actual file anymore!